DataFormats/BTauReco/interface/TemplatedSecondaryVertexTagInfo.h

```diff
@@ -58,7 +58,7 @@ class TemplatedSecondaryVertexTagInfo : public BaseTagInfo {
 
         struct VertexData {
                 VTX                             vertex;
-                Measurement1D                   dist2d, dist3d;
+                Measurement1D                   dist1d,dist2d, dist3d;
                 GlobalVector                    direction;
 		
 		// Used by ROOT storage
```

Comment:
Dear Salva, 
                        I agree the class version need to be updated after change the template thus may need to include an updated checksum. Please do let me know if I have to update classes_def.xml file as well.